### ISSUE TITLE
Add option to auto-confirm deliveries

### ISSUE THREAD

**Issue by morxa** [robocup-logistics/rcll-refbox#38]
It may be useful to auto-confirm deliveries, e.g., when running in simulation.

Add a configuration flag that enables auto-confirmation, which automatically confirms every deliver without checking whether the correct product was delivered.

**Comment by MostafaGomaa** [comment]
Nice Idea, done!
I pushed a commit to PR #30 that implements this behavior.

Worthy to say though, there is a bit of a risk there when the messaging does not function as intended.  For example, if we received an unintended prepare at the DS (which could happen due a systematic error in the command line tools, Or if we received another echo of the same prepare after the machine has gone through all states till IDLE. This could happen especially in simulation )
The effect of that is that we treat this as a new request for a new delivery. This might not be very bad atm as we only have orders of quantity 1 as far as I noticed. So that will only trigger a "too many" delivery.
The worst effect could be on the tracking side though since that new delivery won't be related to a workpiece which will be considered as a failure to read at a DS. In the case that 'fail-safe' is enabled this will cause the workpiece-tracking to be shut down!
